csunplugged/topics/content/en/image-representation/unit-plan/lessons/colour-by-numbers.md

```diff
@@ -0,0 +1,157 @@
+
+{panel type="general"}
+
+# Teaching observations
+
+Possible answers include adding more colours. This is a good answer and colours are part of more advanced lessons in this unit, but this lesson only focusses on black and white pixels, so bring students back to this and ask them how they can make a black and white image more detailed.
+
+{panel end}
+
+The answer you are looking for is to use more pixels for the image, which is what students will do in this next activity.
+
+Put students into groups and give each group a set of the multipage image worksheets, so that each student has a page to work on.
+If this is not possible with the number of students you have just make sure groups are small enough for each students to have at least one page, or preferably for each student to have the same number of pages.
+For example have groups of 3 work on a 6 page image, so each student has 2 pages to complete.
+
+
+Teacher to class: These sets of grids can all be put together to create one larger image, which has much more pixels than the last ones. you are each going to colour in the pixels on your own sheet, in the same way as before, and then put the whole image together.
+
+Once students have coloured in their sheets they can arrange them together to create the whole image.
+There are diagrams on the worksheet printables which teachers can use to help students put their images together if they get stuck.
+
+
+
+## Applying what we have just learnt
+
+- Instead of writing the 1's and 0's into each of the boxes on the grid, have students write out the binary digits that represent their image (in the same way as the 'a' in the lesson starter section) and give this to other students and see if they can re-create the same picture.
+
+- Now that students have tried creating the larger images they could take it a step further and make an image as a whole class.
+The Pixel Painter Parrots option is made up of 32 pages, and is a great challenge for a class.
+
+- Students can also try creating their own pixel art, convert this into a grid of digits, and then have their friends try to re-create their image.
+
+
+{panel type="general"}
+
+# Notes on resources
+
+Students could use grid or graph paper to create their pictures, or a printed grid.
+There is a printable 8 by 8 grid in the printables section.
+
+There are a number of websites students can also use to easily create pixel art.
+After creating their images they can try converting these to grid paper as well.
+
+{panel end}
+
+
+## Lesson reflection
+
+
+This lesson has just covered black and white images, why can't we do more than two colours with this activity?
+
+  - With one binary digit we can only represent two different values.
+  This means if we use one digit to represent the colour of one pixel, then each pixel can only be one of two different colours.
```

Comment:
This should be indented with 4 spaces.

Reply:
Just in some specific parts? Or do all indents need changing?

Reply:
In the Field Guide 2 space indents don't indent, it's 0 spaces to get a list and 4 to indent a sublist. I assume it's the same here so if you want a list to be indented it needs 4 spaces

Reply:
@eAlasdair Just talked to @caitlind. It's purely a style thing but I somehow missed the line above is indented 2 spaces when it should not be. So line 139 will be dedented 2 spaces, and line 140 will remain as is, which puts both lines starting their text from position 3.

Reply:
I thought the conversation was about the line above and wondering why it needed to be indented at all haha